server/routes/chat.go

```diff
@@ -149,3 +143,177 @@ func userInList(a string, list []string) bool {
     return false
 }
 
+func CreateChatRoom(w http.ResponseWriter, r *http.Request) {
+	ctx := appengine.NewContext(r)
+	dsClient := services.Locator.DsClient()
+	userService := services.Locator.UserService()
+	userID := userService.GetCurrentUserID(ctx)
+
+	bodyBytes, err := ioutil.ReadAll(r.Body)
+	if err != nil {
+		log.Printf("Cannot read request body: %v", err)
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+	requestedUserID := string(bodyBytes)
+	id := userID + " " + requestedUserID
+	participants := []string{userID, requestedUserID}
+	cr := models.ChatRoom{ id, participants, nil}
+	
+	chatroomKey := datastore.NameKey("ChatRoom", cr.ID, nil)
+	_, err = dsClient.Put(ctx, chatroomKey, &cr)
+	if err != nil {
+		fmt.Println(err)
+		log.Printf("Cannot create new chat room: %v", err)
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+	
+	//add chatroomKey to CurrentUser.ChatRooms
+	k1 := datastore.NameKey("User", userID, nil)
+	
+	var currentUser models.User
+	if err := dsClient.Get(ctx, k1, &currentUser); err != nil {
+		log.Printf("Cannot update user: %v", err)
```

Comment:
Why are there two `log.Printf`-s inside this `if`? This applies to several `if`-s below as well